site/docs/v1/tech/apis/_two-factor-enable-request-body.adoc

```diff
@@ -37,6 +37,8 @@ You may optionally use the [field]#secretBase32Encoded# value returned by the li
 +
 If [field]#method# is `authenticator` and you omit this field, then [field]#secret# is required.
 
+[field]#twoFactorId# [type]#[String]# [optional]#Optional#::
+Provide this to get a code on the response to complete a Two Factor login.
```

Comment:
Check my example for accuracy, but this is the level of detail we want to provide for the user of this API.

> A two factor identifier that was returned on the Login API when the user is required to configure a two-factor method. When provided, the API response will include a two factor code that can be used to complete a two-factor login request.